packages/mermaid/src/rendering-util/types.ts

```diff
@@ -135,3 +143,8 @@ export type LayoutMethod =
   | 'fdp'
   | 'osage'
   | 'grid';
+
+export interface RenderOptions {
```

Comment:
There is already an interface with the same name. 

https://github.com/mermaid-js/mermaid/blob/4cb0f87b7219ee85afae69102d8c2de81014f0be/packages/mermaid/src/rendering-util/render.ts#L7-L9

Reply:
I'm renaming to `ShapeRenderOptions` for now.